cmd/main.go

```diff
@@ -521,3 +522,8 @@ func setupAndStartOperatorMetadataForwarder(logger logr.Logger, client client.Re
 
 	omf.Start()
 }
+
+func setupAndStartHelmMetadataForwarder(logger logr.Logger, client client.Reader, kubernetesVersion string, options *options, credsManager *config.CredentialManager) {
+	hmf := metadata.NewHelmMetadataForwarder(logger, client, kubernetesVersion, version.GetVersion(), credsManager)
+	hmf.Start()
+}
```

Comment:
Is it possible to call this function again when a DDA is created? 

When the operator is first deployed without credentials or a clusterName, the hmf doesn't get started and it doesn't get started when a valid DDA is created unless the operator pod is deleted and restarted. Not a blocker for this PR, but it would be nicer UX